magma/placer.py

```diff
@@ -141,7 +141,7 @@ def place(self, inst):
         inst.defn = LazyCircuit
         # Setup view now because inline strings might use it during defn
         for sub_inst in getattr(type(inst), "instances", []):
-            setattr(inst, sub_inst.name, InstView(sub_inst, inst))
+            setattr(inst, sub_inst.name, InstView(sub_inst, InstView(inst)))
```

Comment:
Make top-level parent an instview so it's consistent (otherwise we have to check whether it's an instview or inst when walking up the tree)